pkg/systemd/overview-cards/ssh-list-host-keys.sh

```diff
@@ -45,7 +45,7 @@ echo "$systemd" | while IFS='=' read -r name value; do
     fi
 done
 
-keys=$(ssh-keyscan -t dsa,ecdsa,ed25519,rsa -p "$port" "$host" || ssh-keyscan -t ecdsa,ed25519,rsa -p "$port" "$host" || true)
+keys=$(ssh-keyscan -t ecdsa,ed25519,rsa -p "$port" "$host" true)
```

Comment:
You lost the `||` in front of the `true`. This slows down the command severely, as it tries 3 times to resolve the host "true" (which takes some 30 seconds).

Reply:
Woops, tried to quickly push this out yesterday